kp_liberation.Enoch/kp_liberation_config.sqf

```diff
@@ -152,15 +152,15 @@ KP_liberation_preset_civilians = 0;
 9  = CSAT arsenal preset
 10 = Unsung US arsenal preset
 11 = SFP arsenal preset */
-KP_liberation_arsenal = 0;
+KP_liberation_arsenal = 5;
```

Comment:
If we change this to a preset it really limits us as to what mods we can use in arsenal to equip ourselves, at least that is my understanding of it.
Leaving it at 0 should work with what we do in the mission params.